src/main/java/org/tarantool/dsl/Requests.java

```diff
@@ -0,0 +1,109 @@
+package org.tarantool.dsl;
+
+import org.tarantool.dsl.InsertOrReplaceRequestSpec.Mode;
+
+import java.util.List;
+
+/**
+ * Entry point to build requests
+ * using DSL approach a.k.a Request DSL.
+ */
+public class Requests {
```

Comment:
Each request DSL interface offers some kind of request building logic and convenient result processing API instead of function calls with a bunch of parameters. See, for example, how typesafe [JPA Criteria API](https://www.ibm.com/developerworks/ru/library/j-typesafejpa/index.html) is built or take a look at [Querydsl](http://www.querydsl.com/).
IMO, this interface improves select methods a bit but neither is a DSL nor removes the boilerplate completely.